src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -31,7 +66,124 @@ public static void CheckByPassingNullWriter()
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
-        public static void WriteNumber(bool indented)
+        public void WritePropertyOutsideObject(bool skipValidation)
```

Comment:
This test, and the next one, are extracted from the middle of the file (they didn't make sense for JsonDocument... and might not really make sense for JsonElement anymore)